src/lambdaconnect_sync/db_drivers/memory.cljc

```diff
@@ -807,7 +807,8 @@
                          (map t/datomic-name)
                          (set))
          attributes-by-name (-> entity :attributes)
-         sorts (if (seq sorts) sorts [{:key :app/createdAt :direction 1 :options #{}}])
+         sorts (if (seq sorts) sorts [{:key :app/createdAt :direction 1 :options #{}}
+                                      {:key :app/uuid :direction 1 :options #{}}])
```

Comment:
Jak teraz to dokleiłeś, to tak sobie myślę, czy do `sorts`, nawet jeśli jest niepuste, warto dokleić też na końcu sort po uuidach, żeby uniknąć niedeterminizmu.